tests/README.md

```diff
@@ -1,9 +1,8 @@
-IDEA: can go online to determine what things to test
+# Running tests:
 
-To run all tests:
-workon osfoffline
-python -m "nose" .
+1) Make sure pytest is installed: `pip install -r requirements.txt`
+2) Use `py.test` to run the test suite (PyCharm has a run configuration for this also)
```

Comment:
Can I suggest making the preferred command `py.test tests/`?

I'm a bit odd, in that I keep my virtual environment inside my project folder- this command as given runs *everything*. (3500+ tests across all packages)